### PR TITLE
fix Appveyor link in README

### PR DESCRIPTION
**Pull request by StrikerRUS**
image

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
     :target: https://travis-ci.org/scikit-learn-contrib/lightning
 
 .. image:: https://ci.appveyor.com/api/projects/status/mmm0llccmvn5iooq?svg=true
```
Comment (StrikerRUS) [inline README.rst:6]: I guess this link is incorrect as well, but I don't know the right one to fix it.

Actual badge link can be found in the project settings at Appveyor site:

> You can see a badge image URL and sample markup on the **Badges** tab of project settings. 
    https://www.appveyor.com/docs/status-badges/

Comment (mblondel) [reply README.rst:6]: Not entirely sure about this. @fabianp or @vene may have access to the appveyor admin.

Comment (StrikerRUS) [reply README.rst:6]: Hmm... Now I think that **status** link is correct. Looking at the README right now:

image


I don't think that `pending` status can be shown for non-existing project... Let's wait several minutes and if it is correct, it should be green after the merged #167.

Comment (StrikerRUS) [reply README.rst:6]: > ... for non-existing project

If that project was `private` (`... or access denied` part in the message on the site) then without a token in link it cannot be possible to see different statuses right now.

Comment (StrikerRUS) [reply README.rst:6]: Now I'm sure this link is correct:

image

**Pull request merged**